src/kafka_consumer.py

```diff
@@ -2,7 +2,7 @@
 from kafka import KafkaConsumer
 
 # Configuration
-KAFKA_BROKER = 'localhost:9092'  
+KAFKA_BROKER = 'localhost:9092'
 CONSUMER_GROUP_ID = 'social-media-monitor-group'
 TOPICS = ['posts', 'comments', 'emojis']
```

Comment:
Nit for future PR: I believe changing them to ENV value is a better approach, but I understand that it's inthe  initial phase. Including, `KAFKA_BROKER`, `CONSUMER_GROUP_ID`, `TOPICS`